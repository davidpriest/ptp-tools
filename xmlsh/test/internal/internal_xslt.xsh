# test of xslt command

# test identity 
cd ../../samples/styles


xslt -f identity.xsl <<EOF
<foo>
text
</foo>
EOF

# Identity with XML expression


xslt -f identity.xsl -i <[<foo>text</foo>]>

# test complicated style
# Note: includes a comment field which is timestamped so failes test
# xslt -f books.xsl -i ../data/books.xml


