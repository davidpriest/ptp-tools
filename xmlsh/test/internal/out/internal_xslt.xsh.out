<foo>
text
</foo>
<foo>text</foo>
